src/components/SearchBox/SearchBox.tsx

```diff
@@ -1,119 +1,106 @@
-      value,
-      ...props
-    }: Props,
-    ref
-  ) => {
-    const input = React.createRef<HTMLInputElement | null>();
-    const resetInput = () => {
-      onChange && onChange("");
-      if (input.current) {
-        input.current.value = "";
-      }
-    };
-    const submit = (e) => {
-      e.preventDefault();
-      onSubmit && onSubmit(e);
-    };
-    return (
-      <form
-        className={classNames("p-search-box", className)}
-        onSubmit={submit}
-        {...props}
-        ref={ref}
-      >
-        <label className="u-off-screen" htmlFor="search">
-          {placeholder || "Search"}
-        </label>
-        <input
-          autoComplete={autocomplete}
-          className="p-search-box__input"
-          disabled={disabled}
-          id="search"
-          name="search"
-          onChange={(evt) => onChange(evt.target.value)}
-          placeholder={placeholder}
-          ref={input}
-          type="search"
-          defaultValue={externallyControlled ? undefined : value}
-          value={externallyControlled ? value : undefined}
-        />
-        {value && (
-          <button
-            className="p-search-box__reset"
-            disabled={disabled}
-            onClick={resetInput}
-            type="reset"
-          >
-            <Icon name="close">Clear search field</Icon>
-          </button>
-        )}
+const SearchBox = ({
+  autocomplete = "on",
+  className,
+  disabled,
+  externallyControlled,
+  onChange,
+  onSearch,
+  placeholder = "Search",
+  value,
+}: Props): JSX.Element => {
+  const input = React.createRef<HTMLInputElement | null>();
+  const resetInput = () => {
+    onChange && onChange("");
+    if (input.current) {
+      input.current.value = "";
+    }
+  };
+
+  const triggerSearch = () => {
+    onSearch && onSearch();
+  };
+
+  return (
+    <div className={classNames("p-search-box", className)}>
+      <label className="u-off-screen" htmlFor="search">
+        {placeholder || "Search"}
+      </label>
+      <input
+        autoComplete={autocomplete}
+        className="p-search-box__input"
+        disabled={disabled}
+        id="search"
+        name="search"
+        onChange={(evt) => onChange(evt.target.value)}
+        placeholder={placeholder}
+        ref={input}
+        type="search"
+        defaultValue={externallyControlled ? undefined : value}
+        value={externallyControlled ? value : undefined}
+      />
+      {value && (
         <button
-          className="p-search-box__button"
+          className="p-search-box__reset"
           disabled={disabled}
-          type="submit"
+          onClick={resetInput}
+          type="reset"
         >
-          <Icon name="search">Search</Icon>
+          <Icon name="close">Clear search field</Icon>
         </button>
-      </form>
-    );
-  }
-);
+      )}
+      <button
+        className="p-search-box__button"
+        disabled={disabled}
+        type="submit"
```

Comment:
Does it need to be `submit` button? It's not in the form now, and it may cause cases when this button submits forms it's not supposed to submit (when used as part of some other bigger form).